### PR TITLE
Implement groupView service

### PR DESCRIPTION
**Pull request by zenovich**
(contains groupRecentActivity as well)

### CODE REVIEW

File: app/api/groups/get_group.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      | ID | idGroupAncestor | idGroupChild | bIsSelf | iVersion |
+      | 75 | 22              | 13           | 0       | 0        |
+
+  Scenario: The user is an owner of the group
+    Given I am the user with ID "1"
+    When I send a GET request to "/groups/13"
+    Then the response code should be 200
+    And the response body should be, in JSON:
+    """
+    {
```
Comment (smadbe) [inline app/api/groups/get_group.feature]: don't you think we should return the `ID` as well?

File: app/api/groups/get_group.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	groupID, err := service.ResolveURLQueryPathInt64Field(r, "group_id")
+	if err != nil {
+		return service.ErrInvalidRequest(err)
+	}
+
+	user := srv.GetUser(r)
+
+	query := srv.Store.GroupAncestors().OwnedByUserID(user.UserID).
+		Joins("JOIN groups ON idGroupChild=groups.ID").
+		Where("idGroupChild = ?", groupID).Select(
```
Comment (smadbe) [inline app/api/groups/get_group.go]: for the pure logic point of view (so readability as well), I would have expected `groups` as root table. So have `srv.Store.Groups().OwnedBy(user).Where(groups.ID = ?, groupID)`  with the `OwnedBy(...)` doing the "matching," e.g. inner-joining with group ancestors and only caring for the "left" part

File: app/api/groups/get_group.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		`groups.sName, groups.iGrade, groups.sDescription, groups.sDateCreated, groups.sType,
+     groups.sRedirectPath, groups.bOpened, groups.bFreeAccess,
+     groups.sPassword, groups.sPasswordTimer, groups.sPasswordEnd, groups.bOpenContest`).Limit(1)
+
+	var result []map[string]interface{}
+	if err := query.ScanIntoSliceOfMaps(&result).Error(); err != nil {
+		fmt.Println(err)
```
Comment (smadbe) [inline app/api/groups/get_group.go]: println?

**Comment by zenovich** [comment]
> I stop here as it seems there is the other PR included in this one. It makes it very difficult to review. We should either wait for the other PR to be merged or this PR should be on `feat/groupView...<the_other_branch>`

Sorry for this. I thought it's possible since all the task is in the last commit.

File: app/api/groups/get_group.go (hunk 3 above)
Comment (zenovich) [reply app/api/groups/get_group.go]: fixed

File: app/api/groups/get_group.feature (hunk 1 above)
Comment (zenovich) [reply app/api/groups/get_group.feature]: fixed

File: app/api/groups/get_group.go (hunk 2 above)
Comment (zenovich) [reply app/api/groups/get_group.go]: fixed

**Pull request merged**